package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "onboard-notify-react",
-  "version": "0.22.0",
+  "version": "0.23.0",
```

Comment:
In hindsight this should have been `0.22.0-0.1.0` and then we would cut a release branch from develop to merge into develop and master that changes the version to `0.23.0`